src/util.cpp

```diff
@@ -726,17 +728,23 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
                               tr ( "Swedish" ) +
                               "</b></p>"
                               "<p>Daniel (<a href=\"https://github.com/genesisproject2020\">genesisproject2020</a>)</p>"
+                              "<p>tygyh (<a href=\"https://github.com/tygyh\">tygyh</a>)</p>"
+                              "<p>Allan Nordhøy (<a href=\"https://hosted.weblate.org/user/kingu/\">kingu</a>)</p>"
                               "<p><b>" +
                               tr ( "Korean" ) +
                               "</b></p>"
                               "<p>Jung-Kyu Park (<a href=\"https://github.com/bagjunggyu\">bagjunggyu</a>)</p>"
+                              "<p>이정희 (<a href=\"https://hosted.weblate.org/user/MarongHappy/\">MarongHappy</a>)</p>"
```

Comment:
This doesn't show up for me correctly as I don't have support for Korean installed on my machine.

Reply:
Seems ok after I found the correct package.